audioread/rawread.py

```diff
@@ -122,7 +129,9 @@ def duration(self):
         """Length of the audio in seconds (a float)."""
         return float(self._file.getnframes()) / self.samplerate
 
-    def read_data(self, block_samples=1024):
+    def read_data(self, block_samples=None):
+        if block_samples is None:
+            block_samples = self.block_samples
```

Comment:
Please put this after the docstring.

Reply:
Also, an idiomatic way to do this in Python is to use `block_samples = block_samples or self.block_samples`.